templates/cases/inbox_cases.haml

```diff
@@ -29,6 +29,10 @@
             %span.label.label-warning
               [[ item.assignee.name ]]
             &nbsp;
+          %span.label-container{ ng-if:"item.user_assignee.name" }
```

Comment:
Might look better to merge the user and partner into a single string here, e.g. "MOH (Bob Smith)"